includes/classes/SupportMonitor/Monitor.php

```diff
@@ -388,6 +388,44 @@ public function send_message( $data, $type = 'notice', $group = 'message' ) {
 		return true;
 	}
 
+	/**
+	 * Check if loggin is enabled
+	 *
+	 * @return boolean
+	 */
+	public function logging_enabled() {
+		return ( ! defined( 'TENUP_DISABLE_ACTIVITYLOG' ) || ! TENUP_DISABLE_ACTIVITYLOG );
+	}
+
+	/**
+	 * Create a log entry
+	 *
+	 * @param string $summary  Data to log
+	 * @param string $subgroup Sub group
+	 */
+	public function log( $summary, $subgroup = null ) {
+		if ( ! $this->logging_enabled() ) {
+			return;
+		}
+
+		$current_logs = get_option( 'tenup_support_monitor_activity_logs', [] );
+
+		if ( apply_filters( 'tenup_support_monitor_max_activity_log_count', 100 ) <= count( $current_logs ) ) {
```

Comment:
@tlovett1 I question the need to have a limit (or such a small limit). If the option is set to not autoload, the option is able to be up to 4GB in size without any negative effects on the website. I'm thinking if we have a client website that has many users that are doing a lot of updates (like a BuddyPress instance) or a site that registers and creates users regularly, this value of 100 may not be sufficient. Maybe we set it to 500?